docs/dev-guide/edge-agent-dev-guide-renesas-rcar-s4.md

```diff
@@ -0,0 +1,153 @@
+# Getting started with AWS IoT FleetWise Edge Agent on Renesas R-Car S4
+
+This section describes how to deploy AWS IoT FleetWise Edge Agent onto an Renesas [R-Car S4 Reference Board/Spider](https://www.renesas.com/jp/en/products/automotive-products/automotive-system-chips-socs/rtp8a779f0askb0sp2s-r-car-s4-reference-boardspider).
+
+## Prerequisites
+
+- **Renesas Electronics Corporation R-Car S4 Reference Board/Spider**
+  - Spider board has eMMC and micro SD-card slot but the can't use simalutaneously.
+  - For using sdcard, please execute following two steps. The detail files can find in [R-Car S4 SDK Start Up Guide PKG](https://www.renesas.com/us/en/products/automotive-products/automotive-system-chips-socs/r-car-s4-automotive-system-chip-soc-car-servercommunication-gateway#design_development).
+   1. Please flash IPL which support booting from sdcard.
+      - It requires to build IPL with CA_LOAD_TYPE=1 option.
+      - For more details about building IPL, please refer to section 6.2.3 "How to build" in "RENESAS_ICUMX_IPL_for_R-Car_Gen4_Users_Manual_E.pdf"
+   1. Please change SW3 and SW6 on the CPU board.
+      - For more details, please refer to section 3.7.8.1 "Enable SD Card" in "R-Car S4_StartupGuide_x_x_x.pdf"
```

Comment:
Could you please provide the link to the Startup Guide?

Reply:
The link is in line 9. The direct link to the document may change, so the link point to the overview page.